Ten Elephants/Ten Elephants/Controllers/SearchPage/MealSuggestionDataSource.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MealSuggestionDataSource.swift
+//  Ten Elephants
+//
+//  Created by Алексей Шерстнёв on 13.12.2021.
+//
+
+import Foundation
+import UIKit
+
+final class MealSuggestionDataSource: NSObject, UICollectionViewDataSource {
```

Comment:
Разве нужен NSObject?
```suggestion
final class MealSuggestionDataSource: UICollectionViewDataSource {
```

Reply:
Я пробовал убрать NSObject - Xcode на это ругался. Попробую ещё раз

Reply:
![image](https://user-images.githubusercontent.com/43221673/146139517-0d7d75f0-f98c-49b9-8a0a-dd5eac8e83a5.png)